teebbstudios/tuieditor-bundle/1.0/config/packages/teebb_tuieditor.yaml

```diff
@@ -0,0 +1,27 @@
+teebb_tui_editor:
+    #enable: true                           # Whether to enable tui.editor.
+    #jquery: true                           # Whether to enable jquery in dependencies.
+    #jquery_path: ~                         # Custom jquery path.
+    #editor_js_path: ~                      # Custom tui.editor js path.
```

Comment:
We dont need config in comments like this. Rather give a link to your documentation.